config/production.json

```diff
@@ -0,0 +1,9 @@
+{
+  "crmConfig": {
+    "crmBaseUrl": "",
+    "crmUrlPath": "",
+    "portalId": "",
+    "formGuid": "",
+    "subscriptionTypeId": ""
+  }
+}
```

Comment:
I don't think we should check this in... We should `.gitignore` it, with a note in the `README.md` to copy the `default.json` to `production.json` and configure it, instead.
Don't want any chance of accidentally committing "real" credentials from a prod machine.

Reply:
OK sounds good, I'll do this and add a note.

Reply:
Updated and rebased.